doc/project/release_process.rst

```diff
@@ -269,8 +276,9 @@ components provided by the project:
 
 - Compliance with published coding guidelines, style guides and naming
   conventions and documentation of deviations.
-- Regular static analysis on the complete tree using available commercial and
-  open-source tools and documentation of deviations and false positives.
+- Static analysis report
```

Comment:
Since the document is leaving the door open to having multiple code analysis tools (just look at line 280 below, for example), I was under the assumption that there might be several report*S* potentially being made available?

Reply:
ok, fair enough.